internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec_test.go

```diff
@@ -153,13 +188,16 @@ var _ = Describe("Validate clusterlogforwarderspec", func() {
 						Infrastructure: &loggingv1.Infrastructure{}},
 					{Name: "my-audit-logs",
 						Audit: &loggingv1.Audit{}},
+					{Name: "my-external-logs",
+						External: &loggingv1.External{}},
 				},
 			}
 			verifyInputs(forwarderSpec, clfStatus, extras)
 			Expect(forwarderSpec.Inputs).To(HaveLen(3))
 			Expect(clfStatus.Inputs["my-app-logs"]).To(HaveCondition("Ready", true, "", ""))
 			Expect(clfStatus.Inputs["my-infra-logs"]).To(HaveCondition("Ready", true, "", ""))
 			Expect(clfStatus.Inputs["my-audit-logs"]).To(HaveCondition("Ready", true, "", ""))
+			Expect(clfStatus.Inputs["my-external-logs"]).To(HaveCondition("Ready", true, "", ""))
 		})
 
 		It("should validate correctly when input spec defines all three input source specs", func() {
```

Comment:
all four?

Reply:
I dont really know what you mean here, but I have realized that the HaveLen(3) should be HaveLen(4), fix incoming

Reply:
the comment is saying "all three input source specs" and now we have four.

Reply:
ah, got it! Thank you for explaining.